extensions/cornerstone-dicom-sr/src/getSopClassHandlerModule.ts

```diff
@@ -179,7 +176,7 @@ async function _load(displaySet, servicesManager: AppTypes.ServicesManager, exte
   displaySet.isRehydratable = isRehydratable(displaySet, mappings);
   displaySet.isLoaded = true;
 
-  // Check currently added displaySets and add measurements if the sources exist.
+  /** Check currently added displaySets and add measurements if the sources exist */
```

Comment:
This comment should be // as it is a single line comment, not a function/class comment.

Reply:
This is an opinion. I find /** format to be way more cleaner than // which are often use for temporary comments or commenting blocks of code. I use them for "permanent comments" or docs.